File: backend/src/main/java/de/bund/digitalservice/ris/caselaw/domain/TranslationType.java
package de.bund.digitalservice.ris.caselaw.domain;

public enum TranslationType {
  AMTLICH,
  NICHT_AMTLICH,
  KEINE_ANGABE;

  @Override
  public String toString() {
    return switch (this) {
      case AMTLICH -> "amtlich";
      case NICHT_AMTLICH -> "nicht-amtlich";
      case KEINE_ANGABE -> "keine Angabe";
    };
  }
}

File: frontend/src/data/translationType.ts
import {
  TranslationType,
  TranslationTypeLabels,
} from "@/domain/originOfTranslation"

export const translationType = [
  { label: TranslationTypeLabels.AMTLICH, value: TranslationType.AMTLICH },
  {
    label: TranslationTypeLabels.NICHT_AMTLICH,
    value: TranslationType.NICHT_AMTLICH,
  },
  {
    label: TranslationTypeLabels.KEINE_ANGABE,
    value: TranslationType.KEINE_ANGABE,
  },
]
